composer.json

```diff
@@ -54,6 +54,7 @@
         "matthiasnoback/symfony-dependency-injection-test": "^0.7.3",
         "phpunit/phpunit": "^4.8.31",
         "phpspec/prophecy": "^1.4",
+        "symfony/symfony": "^2.8.51 || ^3.0",
```

Comment:
This seems to fix the issue we are currently having in our tests... Altough if I only put it in `require-dev` we have some symfony versions not being tested anymore... should we just raise the minimum version in the real `require` section?